omnisend/class-omnisend-core-bootstrap.php

```diff
@@ -193,6 +199,17 @@ public static function load_omnisend_site_styles(): void {
 	}
 
 	public static function admin_notices(): void {
+		if ( isset( $_SERVER['REQUEST_URI'] ) ) {
+			$request_uri = esc_url_raw( wp_unslash( $_SERVER['REQUEST_URI'] ) );
+			if ( strpos( $request_uri, '/wp-admin/admin.php?page=omnisend' ) !== false ) {
+				remove_all_actions( 'admin_notices' );
+				if ( self::is_hostinger_plugin_active() ) {
+					echo '<div class="omnisend-custom-notice"><img src="' . esc_url( plugin_dir_url( __FILE__ ) ) . 'assets/img/omnisend-notice-discount-icon.svg" />
```

Comment:
Could we add this via react to avoid frontend logic split between react and php?